code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -173,6 +173,12 @@
 	icon_state = "bottleofnothing"
 	list_reagents = list("nothing" = 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/bottleofbanana
+	name = "Jolly Jug"
```

Comment:
Currently, all bottles, including the Bottle of Nothing is capitalized, making it lower-case would make it inconsistent.